docs/en/ingest-management/elastic-agent/elastic-agent-configuration.asciidoc

```diff
@@ -78,8 +80,11 @@ are ignored.
 
 To enable monitoring, set `agent.monitoring.enabled` to `true`. Also set the
 `logs` and `metrics` settings to control whether logs, metrics, or both are
-collected. If neither setting is specified, monitoring is disabled. Set
-`use_output` to specify the output to which monitoring events are sent.
+collected. If neither setting is specified, monitoring is disabled. To
+disable the {beats} from emmitting metrics entries into their logs, set
+`agent.monitoring.log_metrics` to `false`, this will not effect the `metrics`
+setting.  Set `use_output` to specify the output to which monitoring events are
```

Comment:
```suggestion
`agent.monitoring.log_metrics` to `false`. This will not affect the `metrics`
setting. Set `use_output` to specify the output to which monitoring events are
```

Reply:
Fixes comma splice and word usage.